pkg/sync/v7/data/ConfigMap/openshift-azure-logging/container-azm-ms-agentconfig.yaml

```diff
@@ -0,0 +1,292 @@
+apiVersion: v1
+data:
+  config-version: ver1
+  kube.conf: |
+    # Fluentd config file for OMS Docker - cluster components (kubeAPI)
```

Comment:
IMO it would be good to include a link somewhere (commit body or comment) to documentation on how these configs were generated.

Reply:
There's a ruby script inside the agent container which is responsible for generating default configs in the event that a wrong config or none was provided. This config was generated by running the agent with no config, letting the ruby script inside the agent image generate the config and then I copied that content.